src/room_member.cpp

```diff
@@ -231,11 +232,17 @@ void RoomMember::ReceiveLoop() {
             }
 
             peer->DeallocatePacket(packet);
+            if(!IsConnected())      //Don't parse any more packets if we should shutdown
```

Comment:
I removed this check, but it could be necessary to have something like if (state == State::Leave) shutdown=true; to assure that the inner loop doesn't continue if the member continues to receive packets.